CHANGES.rst

```diff
@@ -4,6 +4,10 @@ Change History
 2.5.3 (unreleased)
 ==================
 
+- In the tests, set ``TERM=dumb`` in the environment.  This avoids
+  invisible control characters popping up in some terminals, like
+  ``xterm``.  [maurits]
+
```

Comment:
I suggest clarifying that this happens under the ' system()'  function used in tests.